src/pages/_app.tsx

```diff
@@ -1,20 +1,53 @@
-import { FC } from 'react';
+import { createContext, FC, useEffect, useState } from 'react';
 import type { AppProps } from 'next/app';
 import { ToastContainer } from 'react-toastify';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
 import 'react-toastify/dist/ReactToastify.css';
 import '@/styles/index.scss';
-import AppWrapperContext from '@/context'
+import AppWrapperContext from '@/context';
+import fetch from '@/helperFunctions/fetch';
 
 if (process.env.NEXT_PUBLIC_API_MOCKING === 'ON') {
   require('../../__mocks__/mswInit');
 }
 
-const MyApp: FC<AppProps> = ({ Component, pageProps }) => (
-  <AppWrapperContext>
-    <ToastContainer />
-    <Component {...pageProps} />
-  </AppWrapperContext>
-);
+const SELF_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/self`;
+const { ERROR } = ToastTypes;
+export const isUserAuthorizedContext = createContext<Boolean>(false);
+
+const MyApp: FC<AppProps> = ({ Component, pageProps }) => {
```

Comment:
Can we create another file to create this context. I don't think we should put this much code in _app.tsx ? Check AppWrapperContext for reference

Reply:
I am using isUserAuthorized file for the context, and using it here importing it.

Reply:
@Maheima i am using isUserAuthorized file for context and using it by importing here.

Reply:
Awesome!